### DISCUSSION TITLE
How to manage track order

### DISCUSSION THREAD

**Discussion by AlbatrosHurleur** [GMOD/jbrowse-components#3048] [Q&A] [answered]
Hello,

I am wondering how the tracks are ordered and if this order can be manually modified?

For instance, in this case the order is not alphabetical and doesn't follow config.json order (where `H3K27me3 (peaks)` is right after `H3K27me3 (logFoldEnrichment)`).

![image](https://user-images.githubusercontent.com/48366457/174786654-e8f9fa5d-d13b-4619-854d-055463f27c21.png)

Kind regards

**Comment by cmdcolin** [comment, marked as answer]
good question, I think we actually don't have anything for this really. it is semi-related to config.json but if you have categories it may have different orderings. It might be worth it for us to formalize alphabetical and/or custom orderings. open issue for this https://github.com/GMOD/jbrowse-components/issues/26

**Reply by AlbatrosHurleur** [reply]
Yes that would be a nice feature!
Indeed I have categories and some sub-categories.
